src/meshapi/management/commands/scramble_members.py

```diff
@@ -45,6 +45,7 @@ def handle(self, *args: Any, **options: Any) -> None:
                 member.stripe_email_address = ""
                 member.additional_email_addresses = []
                 member.phone_number = fake.phone_number()
+                member.additional_phone_numbers = []
```

Comment:
Might be good to generate a few randomly.

Reply:
Done